static/js/modules/send-message.js

```diff
@@ -6,10 +6,6 @@ var _ = require('underscore'),
 
   'use strict';
 
-  var translateFn = function(key) {
```

Comment:
I looked into it, it was added by @garethbowen in bf457cc1593396790e61bc36cc39bc93ba13374f , and I can't see any particular reason for its existence in that file. The idea might have been for testing later, but I removed it and all the tests pass, so if that was ever an issue it is no longer one.